### DISCUSSION TITLE
A transparent mode

### DISCUSSION THREAD

**Discussion by emsi** [langfuse/langfuse#2697] [Ideas]
### Describe the feature or potential improvement

As OpenAI API becomes a de-facto industry standard, it would be great if one could use langfuse simply by changing the base_url and point to langfuse server.
It might require configuring available models on langfuse but it would also enable tracing and collecting metrics for 3rd party apps that cannot be modified or supporting unforeseen languages and frameworks.

**Comment by marcklingen** [comment]
I'd recommend to check out the litellm proxy for this. It is natively integrated with Langfuse to log out llm calls and standardizes lots of models on the openai api schema.

Until now it has been a core design decision not to be in the critical path for Langfuse Tracing but rather log data to langfuse asynchronously.

Docs: https://langfuse.com/docs/integrations/litellm/tracing
